app/handlers/testConfig.py

```diff
@@ -20,4 +31,61 @@ class TestConfigHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
-        self.finish("Test feature tracking")
\ No newline at end of file
+
+        identifier = self.request.body_arguments["identifier"]
+        test_flag = self.request.body_arguments["test_flag"]
+
+        try:
+            frame_start = int(self.request.body_arguments["frame_start"])
+            num_frames = int(self.request.body_arguments["num_frames"])
+        except Exception as e:
+            frame_start = 0
+            num_frames = 120
+            print e
+
+        if test_flag == "feature":
+            runConfigTestFeature(identifier, frame_start, num_frames)
+        if test_flag == "object":
+            runConfigTestObject(identifier, frame_start, num_frames)
+
+        message = "Hello,\n\tWe have finished processing your video and identifying all objects.\nThank you for your patience,\nThe Santos Team"
+        subject = "Your video has finished processing."
+
+        EmailHelper.send_email(self.request.body_arguments["email"], subject, message)
+        self.finish("Test feature tracking")
+
+    def runConfigTestFeature(self, identifier, frame_start, num_frames):
+        project_path = get_project_path(identifier)
+
+        tracking_path = os.path.join(ac.CURRENT_PROJECT_PATH, ".temp", "test", "test_feature", "feature_tracking.cfg")
+        db_path = os.path.join(ac.CURRENT_PROJECT_PATH, ".temp", "test", "test_feature", "test1.sqlite")
+        if os.path.exists(db_path):
+            os.remove(db_path)
+
+        images_folder = "feature_images"
+        video.delete_images(images_folder)
+
+        subprocess.call(["feature-based-tracking", tracking_path, "--tf", "--database-filename", db_path])
+        subprocess.call(["display-trajectories.py", "-i", get_project_video_path(identifier), "-d", db_path, "-o", ac.CURRENT_PROJECT_PATH + "/homography/homography.txt", "-t", "feature", "--save-images", "-f", str(frame_start), "--last-frame", str(frame_start+num_frames)])
+
+        video.move_images_to_project_dir_folder(images_folder)
```

Comment:
I now remember. The reason we wanted to generate images before was that our Qt was broken and couldn't play videos. I think that the best API design would be to use `video` functions to create and return a video. We'll decompose that into frames on the client side.